pkg/skaffold/build/gcb/docker.go

```diff
@@ -54,11 +61,27 @@ func (b *Builder) dockerBuildSpec(a *latest.Artifact, tag string, platforms plat
 	}, nil
 }
 
+func platformEmulatorInstallStep(cfg *latest.GoogleCloudBuild, p platform.Matcher) []*cloudbuild.BuildStep {
```

Comment:
Is it possible to return `*cloudbuild.BuildStep` instead of `[]*cloudbuild.BuildStep` taking into account that in line 46 we are using `...` to do the append?